library/pk_wrap.c

```diff
@@ -716,4 +720,214 @@ const mbedtls_pk_info_t mbedtls_rsa_alt_info = {
 
 #endif /* MBEDTLS_PK_RSA_ALT_SUPPORT */
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+
+static void *pk_psa_alloc_wrap( void )
+{
+    void *ctx = mbedtls_calloc( 1, sizeof( psa_key_slot_t ) );
+
+    if( ctx != NULL )
+        memset( ctx, 0, sizeof( psa_key_slot_t ) );
```

Comment:
Side-question: Standard `calloc()` clears the content of the newly allocated buffer - do we require that for potential replacements, too?

Reply:
I think we do. Tbh, I'm not entirely sure why we have that code block here - I think it's for uniformity with the case where we have an `_init()` function to call, but I'm not sure that's a good reason. Wdyt?

Reply:
Replacing that call with a comment.